### PR TITLE
Adding screenshots to the README

### PR DESCRIPTION
**Pull request by pshriwise**
It was aptly out to me that there aren't any screenshots in the README of a project with a GUI. This PR adds several screenshots of key features that users may find particularly interesting.

@friedmud 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,15 @@
 
 ## Description
 
-A graphical interface for visualizing and manipulating OpenMC plot slices.
+A graphical interface for visualizing and navitaging OpenMC models.
```
Comment (paulromano) [inline README.md]:
```suggestion
A graphical interface for visualizing and navigating OpenMC models.
```

File: README.md [hunk 2]
```diff
@@ -21,18 +25,57 @@ This package is most easily installed by running
 pip install openmc-plotter
 ```
 
-
 ## Usage
 
-From a directory containing a working OpenMC model run:
+From a directory containing an OpenMC model run:
```
Comment (paulromano) [inline README.md:30]: What, you mean is doesn't work from a directory containing a broken model?! :laughing:

**Review by paulromano — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pshriwise) [reply README.md:30]: Haha yeah. I felt like that didn't need to be specified.

**Review by paulromano — APPROVED** [review]

**Comment by friedmud** [comment]
Hah - they look great!

**Pull request merged**